Select.js

```diff
@@ -290,10 +291,11 @@ define([
 			}
 		},
 		
-		getIdentity: function (dataItem) {
-			// Override of delite/Selection's method
-			return this.store.getIdentity(dataItem);
-		},
+		getIdentity: dcl.superCall(function (sup) {
+            return function (dataItem) {
+                return sup.call(this, dataItem);
```

Comment:
same tabbing/alignement issue. Please check it everywhere (isn't jshint raising errors?)